onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -7592,7 +7592,7 @@ def aten_scalar_tensor_sym_number(
     return common_ops.cast_to(s, dtype=dtype)
 
 
-@torch_op("aten::scatter.value", trace_only=True)
+@torch_op(("aten::scatter.value", "aten::scatter.src"), trace_only=True)
```

Comment:
Looks like scatter.value is taking src as a single value. Maybe we don't need to expand in this case?